### DISCUSSION TITLE
How to import CSS files and images?

### DISCUSSION THREAD

**Discussion by faressoft** [LaunchMenu/LaunchMenu#187] [General] [answered]
I defined my React component in a file with .tsx extension to be used as content for the items of the menu. Does lm-build-tools resolve importing css and images?

```tsx
import './style.css'
import logo from './img/logo.png';
```

**Comment by TarVK** [comment, marked as answer]
No, neither of these things are setup by default. 

We usually use [emotionjs](https://emotion.sh/docs/introduction) for styling, through our own wrapper class, see [theming](https://launchmenu.github.io/docs/guide/5.-content-and-theming#theming).
For images and other resources we usually use internal relative paths, e,g.: [ShortAboutLM.ts](https://github.com/LaunchMenu/LaunchMenu/blob/development/packages/core/src/application/components/ShortAboutLM.tsx#L27)

The build-tools don't rely on a bundler, which makes adding such behavior a bit more difficult. Instead, all "static files" are copied across to the output so you can refer to them with a relative path. The default file extensions this works for is defined here: [build-tools/index.js](https://github.com/LaunchMenu/LaunchMenu/blob/development/packages/dev-tools/build-tools/index.js#L35)

I understand this might not be ideal if your normal workflow is different. It's unlikely we will add these alternative imports into our build tools, since I think it would require significant changes. You can however always try to setup your only build process that works according to your needs, there's no real need to use ours.

**Reply by faressoft** [reply]
Thank you very much
